src/coprocessor/dag/executor/scan.rs

```diff
@@ -170,6 +186,10 @@ impl<S: Store, T: InnerExecutor> Executor for ScanExecutor<S, T> {
         }
     }
 
+    fn collect_execution_summaries(&mut self, target: &mut [ExecSummary]) {
```

Comment:
How about using one function like `collect_statistics` in batch?

Reply:
It may involve some refinement for existing statistics, which may be not related to this PR. We can do it in future PRs.